templates/apps/sumo-aws-apps/security-hub-org/security-hub-org-main.yaml

```diff
@@ -222,71 +201,31 @@ Parameters:
     MinLength: 12
     MaxLength: 12
 
-  EnableCISStandard:
+  SecurityStandards:
     Type: String
```

Comment:
may we can use CommaDelimitedList type https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/parameters-section-structure.html

Reply:
okay

Reply:
Error: Value of property Parameters must be an object with String (or simple type) properties .https://github.com/aws-cloudformation/cfn-language-discussion/issues/115. we need use type string